src/renderers/dom/shared/DOMMarkupOperations.js

```diff
@@ -85,13 +88,16 @@ var DOMMarkupOperations = {
    * @param {*} value
    * @return {?string} Markup string, or null if the property was invalid.
    */
-  createMarkupForProperty: function(name, value) {
+  createMarkupForProperty: function(name, value, namespace) {
     var propertyInfo = DOMProperty.getPropertyInfo(name);
     if (propertyInfo) {
       if (shouldIgnoreValue(propertyInfo, value)) {
         return '';
       }
       var attributeName = propertyInfo.attributeName;
+      if (namespace === HTML_NAMESPACE) {
+        attributeName = attributeName.toLowerCase();
```

Comment:
Should this happen once, when the properties are injected?

Reply:
That wouldn't help with properties that aren't in the whitelist.

Reply:
But line 93 ensures that this code only runs when we have property info (which I believe is always from the whitelist)

Reply:
Ah I see. I guess we should update the custom ones too in this case.

Reply:
Wait. Why is this necessary? This already gets lower cased in the injection here:
https://github.com/facebook/react/blob/master/src/renderers/dom/shared/DOMProperty.js#L83

Reply:
Looks like tests pass when I remove this statement! (98-100). Totally need the `toLowerCase` below though.